src/herbpy/action/actions.py

```diff
@@ -0,0 +1,145 @@
+import numpy, openravepy, prpy, time
+from prpy import tsr
+from prpy.tsr import tsrlibrary
+from prpy.tsr.tsr import *
+
+@ActionMethod
+def MoveObject(robot, direction = [0, 0, 1], distance = 0.1):
```

Comment:
You should [*never* use a mutable object (a `list` in this case) as a default value in Python](http://stackoverflow.com/q/1132941/111426). Please replace this with `direction=None` and set the default inside the function body if `direction is None`.